packages/medusa/src/api/admin/product-options/route.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+  refetchEntities,
+  refetchEntity,
+} from "@medusajs/framework/http"
+
+import { createProductOptionsWorkflow } from "@medusajs/core-flows"
+import { HttpTypes } from "@medusajs/framework/types"
+
+export const GET = async (
+  req: AuthenticatedMedusaRequest<HttpTypes.AdminProductOptionListParams>,
+  res: MedusaResponse<HttpTypes.AdminProductOptionListResponse>
+) => {
+  const { data: product_options, metadata } = await refetchEntities({
```

Comment:
suggestion: this `refetchEntities` will also disapear in the future, if we can rely on the graph as well for those it would be awesome 💪